### PR TITLE
Iosxe/show wireless client summary

### PR DESCRIPTION
**Pull request by itdependsnetworks**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
+import re
```
Comment (badalsoyantar) [inline src/genie/libs/parser/iosxe/show_wireless.py:1]: Write all parser commands on top as comment

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        #
+        # Number of Excluded Clients: 2
+        #
+        # MAC Address    AP Name                          Type ID   State              Protocol Method
+        # ------------------------------------------------------------------------------------------------
+        # 58bf.ea8a.20e6 b80-61-cap20                   WLAN 17   Excluded           11ac     Dot1x
+        # 58bf.ea37.4442 b80-62-cap14                   WLAN 17   Excluded           11ac     Dot1x
+
+        # Number of Clients: 13
+        wireless_client_count_capture = re.compile(r"^Number\s+of\s+Clients:\s+(?P<wireless_client_count>\d+)$")
```
Comment (badalsoyantar) [inline src/genie/libs/parser/iosxe/show_wireless.py:89]: usually we use p1,p2 and so on as variable name for regex pattern as it is easy to track.

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+            # Number of Clients: 13
+            if wireless_client_count_capture.match(line):
+                wireless_client_count_capture_match = wireless_client_count_capture.match(line)
+                groups = wireless_client_count_capture_match.groupdict()
+                if not show_wireless_client_summary_dict.get('wireless_client_summary', {}):
+                    show_wireless_client_summary_dict['wireless_client_summary'] = {}
+                wireless_client_count = int(groups['wireless_client_count'])
+                show_wireless_client_summary_dict['wireless_client_summary'][
+                    'wireless_client_count'] = wireless_client_count
+                continue
```
Comment (badalsoyantar) [inline src/genie/libs/parser/iosxe/show_wireless.py:126]: if you are using elif no need for continue? As it will continue loop anyway?

**Review by badalsoyantar — CHANGES_REQUESTED** [review]

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 2 above)
Comment (itdependsnetworks) [reply src/genie/libs/parser/iosxe/show_wireless.py:89]: Spoke with @jeaubin agreed to leave these as is.

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 3 above)
Comment (itdependsnetworks) [reply src/genie/libs/parser/iosxe/show_wireless.py:126]: This is to support this design pattern https://github.com/CiscoTestAutomation/genieparser/pull/256#discussion_r493013132 e.g. a future raise after this.

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 2 above)
Comment (itdependsnetworks) [reply src/genie/libs/parser/iosxe/show_wireless.py:89]: For this and future cases.

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 1 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/show_wireless.py:1]: Can you clarify what you mean? Do you mean this comment block? 
```
# =================================
# Schema for:
#  * 'show wireless client summary'
# =================================
```

Comment (badalsoyantar) [reply src/genie/libs/parser/iosxe/show_wireless.py:1]: yes

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 2 above)
Comment (badalsoyantar) [reply src/genie/libs/parser/iosxe/show_wireless.py:89]: confirmed

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 3 above)
Comment (badalsoyantar) [reply src/genie/libs/parser/iosxe/show_wireless.py:126]: confirmed

**Review by badalsoyantar — APPROVED** [review]

**Review by ThomasJRyan — APPROVED** [review]

**Comment by ThomasJRyan** [comment]
Please resolve the merge conflicts

**Pull request merged**